lstchain/calib/camera/flatfield.py

```diff
@@ -118,20 +118,20 @@ def calculate_relative_gain(self, event):
         if self.num_events_seen == self.sample_size:
             self.num_events_seen = 0
 
+        pixel_mask = np.logical_or(
+            event.mon.tel[self.tel_id].pixel_status.hardware_failing_pixels,
```

Comment:
how do you identify `hardware_failing_pixels `?

Reply:
no, I get them from the EvB